src/sparseml/pytorch/image_classification/train.py

```diff
@@ -672,14 +676,21 @@ def train(
     :param rank: The rank of the process
     """
 
+    val_res = None
     if not trainer.one_shot:
         # Baseline eval run
-        trainer.run_one_epoch(
-            mode="validation",
+        val_res = trainer.run_one_epoch(
+            mode="val",
             max_steps=max_eval_steps,
             baseline_run=True,
         )
-    val_res = None
+
+        LOGGER.info(f"\nInitial validation results: {val_res}")
+
+        eval_results_path = os.path.join(save_dir, "eval.txt")
+        helpers.write_validation_results(eval_results_path, val_res, epoch=0)
+        print(f"Saving initial validation results to {eval_results_path}")
```

Comment:
nit: let's move this to an info log and move the log to the helper function (remove word initial)